### DISCUSSION TITLE
How about adding #define, #ifdef etc preprocessing?

### DISCUSSION THREAD

**Discussion by cpsource** [modular/modular#230] [Ideas]
I do sorely miss this feature from 'c'. But I suppose I could run cpp over the python first. Still...

**Comment by lattner** [comment]
We're not really looking to add "general goodness" features to mojo at this point vs Python, I'd encourage you to bring it up with the python PEP process. That said, I expect Mojo's metaprogramming features to define away many reasons for #ifdefs.
